README.md

```diff
@@ -14,17 +14,91 @@ The `ballerinax/spotify` package provides APIs to connect and interact with Spot
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Spotify connector, you must have access to the Spotify API through a [Spotify developer account](https://developer.spotify.com) and a project under it. If you do not have a Spotify Developer account, you can sign up for one [here](https://developer.spotify.com/documentation/web-api/concepts/access-token).
+
+### Step 1: Create a Spotify Developer Project
+
+1. Open the [Spotify Developer Portal](https://developer.spotify.com/dashboard).
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/dashboad.png alt="Dashboad" style="width: 70%;">
+
+
+2. Click on the "Create an app" tab and select an existing project or create a new one for which you want API Keys and Authentication Tokens.
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/app_overview.png alt="Spotify Developer Portal" style="width: 70%;">
+
+
+### Step 2: Set up user authentication settings
```

Comment:
As a convention we use sentence case for all the headings in documentation. Please apply same for all the headers
```suggestion
### Step 1: Create a Spotify developer project

1. Open the [Spotify Developer Portal](https://developer.spotify.com/dashboard).
<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/dashboad.png alt="Dashboad" style="width: 70%;">


2. Click on the "Create an app" tab and select an existing project or create a new one for which you want API Keys and Authentication Tokens.
<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/app_overview.png alt="Spotify Developer Portal" style="width: 70%;">


### Step 2: Set up user authentication settings
```